src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.java

```diff
@@ -110,26 +110,25 @@ protected int countCollectorFilesGeneratedFromSAEFilesToday() {
         return searchResults.size();
     }
 
-    protected String writeToCollectorFile(String originalFileName, CollectorBatch collectorBatch,
-            ConcurStandardAccountingExtractBatchReportData reportData) {
-        String collectorFilePath = buildCollectorFilePath(originalFileName);
+    protected String writeToCollectorFile(String originalFileName, CollectorBatch collectorBatch) {
+        String collectorFileName = buildCollectorFileName(originalFileName);
+        String collectorFilePath = buildCollectorFilePath(collectorFileName);
         boolean fileCreatedSuccessfully = collectorFlatFileSerializerService.serializeToFlatFile(collectorFilePath, collectorBatch);
         if (!fileCreatedSuccessfully) {
             LOG.error("writeToCollectorFile(): An error occurred while writing the data to the Collector file; see earlier logs for details.");
-            reportData.addHeaderValidationError("Encountered an unexpected I/O error when generating the Collector file");
             return StringUtils.EMPTY;
         }
-        return collectorFilePath;
+        return collectorFileName;
     }
 
-    protected String buildCollectorFilePath(String saeFileName) {
+    protected String buildCollectorFileName(String saeFileName) {
         String saeFileNameWithoutExtension = StringUtils.substringBeforeLast(saeFileName, KFSConstants.DELIMITER);
-        return new StringBuilder(DEFAULT_BUILDER_SIZE)
-                .append(collectorDirectoryPath)
-                .append(ConcurConstants.COLLECTOR_CONCUR_OUTPUT_FILE_NAME_PREFIX)
-                .append(saeFileNameWithoutExtension)
-                .append(GeneralLedgerConstants.BatchFileSystem.EXTENSION)
-                .toString();
+        return ConcurConstants.COLLECTOR_CONCUR_OUTPUT_FILE_NAME_PREFIX + saeFileNameWithoutExtension
+                + GeneralLedgerConstants.BatchFileSystem.EXTENSION;
+    }
+
+    protected String buildCollectorFilePath(String collectorFileName) {

```

Comment:
should this function buildCollectorFileFullyQualifiedFileName?  We are using the "fullyQualified" naming convention when the file name includes the path.